pkg/configuration/configuration.go

```diff
@@ -50,8 +48,13 @@ const (
 
 	ClusterUnavailableDelay        = "cluster.unavailable.delay"
 	DefaultClusterUnavailableDelay = "60s"
+
+	identityProviderName        = "identity.provider"
+	DefaultIdentityProviderName = "rhd"
```

Comment:
it's just a detail - not sure why you move it - you put the constants to the part containing KubeFed configuration constants

Reply:
fixed here: cecef2d2b60d85e6e54793c4d68db1168ee8c449

Reply:
@MatousJobanek i just noticed that all the kubefed vars are exported - is there any reason for this? i just checked and it doesn't look like they are used anywhere other than in configuration. if there is no reason, i can fix this in this PR

Reply:
You can keep it there - I'm going to remove most of the constants in the next PRs anyway.